deploy/kubernetes/setup-csi-snapshotter.yaml

```diff
@@ -89,9 +89,15 @@ spec:
           args:
             - "--csi-address=$(ADDRESS)"
             - "--connection-timeout=15s"
+            - "--leader-election=true"
+            - "--leader-election-namespace=$(POD_NAMESPACE)"
```

Comment:
Actually, do we need to set the namespace?

Reply:
nope, updating